api-controllers/DashboardController.js

```diff
@@ -277,4 +277,31 @@ function manageInstancePath(session) {
   return session.instance_type ? `/manage/dashboards/${session.instance_type}/instances/${session.instance_id}` : `/manage/instances/${session.service_id}/${session.plan_id}/${session.instance_id}`;
 }
 
+function createService(plan_id, instance_id, context) {
+  const plan = catalog.getPlan(plan_id);
+  const options = {
+    plan_id: plan_id,
+    context: context
+  };
+  switch (plan.manager.name) {
+
+  case CONST.INSTANCE_TYPE.DIRECTOR:
+    return DirectorService.createInstance(instance_id, options);
+
```

Comment:
in our current code we dont have newlines between cases. Please remove unnecessary lines.